internal/service/apigateway/domain_name.go

```diff
@@ -416,10 +472,43 @@ func resourceDomainNameDelete(ctx context.Context, d *schema.ResourceData, meta
 	return diags
 }
 
-func findDomainByName(ctx context.Context, conn *apigateway.Client, domainName string) (*apigateway.GetDomainNameOutput, error) {
+const domainNameResourceIDSeparator = "/"
```

Comment:
Can a comma delimiter work here to keep in line the with standard `flex.[Expand|Flatten]ResourceID` helper functions?

https://github.com/hashicorp/terraform-provider-aws/blob/42a5eedc7a92d5e36f394b0231ee657c9b5fab84/internal/flex/flex.go#L251-L253

https://github.com/hashicorp/terraform-provider-aws/blob/42a5eedc7a92d5e36f394b0231ee657c9b5fab84/internal/flex/flex.go#L22-L25

Reply:
Yes, it would -- I'll change

Reply:
Wasnt familiar with this, will use it from now on